syntax = "proto3";

package mprpc;

#开启生成通用服务代码的选项
option cc_generic_services = true; 

message RpcHeader
{
    bytes service_name = 1;
    bytes method_name = 2;
    uint32 args_size = 3;
}